bundles/binding/org.openhab.binding.expire/src/main/java/org/openhab/binding/expire/internal/ExpireBinding.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.expire.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openhab.binding.expire.ExpireBindingProvider;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This binding monitors state changes and sets the state to "Undefined" if not state change occurs within
+ * the configured time
+ *
+ * @author Michael Wyraz
+ * @since 1.9.0
+ */
+public class ExpireBinding extends AbstractActiveBinding<ExpireBindingProvider>
+{
+
+    private static final Logger logger = LoggerFactory.getLogger(ExpireBinding.class);
+
+    /**
+     * The BundleContext. This is only valid when the bundle is ACTIVE. It is set in the activate()
+     * method and must not be accessed anymore once the deactivate() method was called or before activate()
+     * was called.
+     */
+    private BundleContext bundleContext;
+
+    /**
+     * the refresh interval which is used to poll values from the Expire
+     * server (optional, defaults to 60000ms)
+     */
+    private long refreshInterval = 1000;
```

Comment:
comment disagrees with the default value.  Also, I don't think the current code has a way to change it via configuration (not that I can think of any reason to make it configurable), so I think the comment should be improved.